lang/en.json

```diff
@@ -777,67 +796,67 @@
+        "ZendeskChatTagHelp": "This tag allows adding the Zendesk Chat form to your website.",
         "ZendeskChatTagName": "Zendesk Chat (formerly Zopim)",
-        "SettingElementVisibilityObserveDomChangesTitle": "Observe DOM Changes",
-        "SettingElementVisibilityObserveDomChangesDescription": "Enable if you want to track dynamic DOM elements. %1$sNote: This might slow down the website if there are lot of DOM changes%2$s",
+        "SettingElementVisibilityObserveDomChangesDescription": "Allows tracking dynamic DOM elements. %1$sNote: May slow down the website if there are many DOM changes%2$s",
+        "EntityRecursionExceptionForVariable": "An entity references itself, or a recursion is caused in some other way. Possibly related due to this entry: \"%1$s\"",
         "EntityRecursionExceptionForVariable": "It seems an entity references itself or a recursion is caused in some other way. It may be related due to this entry: \"%1$s\"",
         "AllDownloadsClickTriggerDownloadExtensionsTitle": "Download Extensions",
-        "AllDownloadsClickTriggerDownloadExtensionsDescription": "Comma separated list of file extensions which will be considered as a download.",
-        "CustomEventTriggerEventNameDescription": "The name of the event that is pushed to the Data-Layer. For example you can push an event by adding this to your website: _mtm.push({\"event\": \"my-custom-event\"});",
+        "AllDownloadsClickTriggerDownloadExtensionsDescription": "Comma-separated list of file extensions considered as a download.",
+        "CustomEventTriggerEventNameDescription": "The name of the event that is pushed to the dataLayer. E.g. you can push an event by adding this to your website: _mtm.push({\"event\": \"my-custom-event\"});",
         "ElementVisibilityTriggerSelectionMethodTitle": "Selection Method",
         "ElementVisibilityTriggerSelectionMethodDescription": "Select the way you want to identify an element you want to select.",
         "ElementVisibilityTriggerCssSelectorTitle": "CSS Selector",
         "ElementVisibilityTriggerCssSelectorDescription": "A CSS selector allows you to select an element by id, className, element names, etc. If multiple elements match this selector, the first matching element will be used to get the value from. Examples for valid selectors are \".classname\", \"#id\" or \"li a\".",
         "ElementVisibilityTriggerElementIDTitle": "Element ID",
-        "ElementVisibilityTriggerElementIDDescription": "The id attribute specifies a unique id for an HTML element. Insert here the value of an ID attribute of any element within your website.",
-        "ElementVisibilityTriggerFireTriggerWhenTitle": "Fire this trigger",
+        "ElementVisibilityTriggerElementIDDescription": "The ID attribute specifies a unique id for an HTML element. Insert here the value of an ID attribute of any element within your website.",
+        "ElementVisibilityTriggerFireTriggerWhenTitle": "Trigger this trigger",
```

Comment:
@comradekingu People may be used to "Fire this trigger" terminology as its similar from GA

Reply:
```suggestion
        "ElementVisibilityTriggerFireTriggerWhenTitle": "Fire this trigger",
```